src/main/java/core/basesyntax/service/PurchaseOperationHandler.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+
+public class PurchaseOperationHandler implements OperationHandler {
+    private FruitDao fruitDao = new FruitDaoImpl();
```

Comment:
The code should not be implementation-dependent 
use a constructor for initialization
```suggestion
    private FruitDao fruitDao;
    
    public PurchaseOperationHandler(FruitDao fruitDao) {
          this.fruitDao = fruitDao;
    }
```